### DISCUSSION TITLE
Add parameter to build_data/context.xml - resource name

### DISCUSSION THREAD

**Discussion by jesade-vbg** [kartoza/docker-geoserver#705] [Ideas]
A little suggestion.
In the content.xml template is should be very useful to ba able to change the Resource name
https://github.com/kartoza/docker-geoserver/blob/df6bc9e2d76dea1f375771701cc8801bdbba0c46/build_data/context.xml#L19

![image](https://github.com/user-attachments/assets/2c152420-aa37-4d03-8039-e629209b1283)

For example
`<Resource name="jdbc/${POSTGRES_JNDI_NAME}"`

(the default value should of course be "postgres" as before)

**Comment by NyakudyaA** [comment]
@jesade-vbg You are welcome to do a PR with the suggested change

**Comment by jesade-vbg** [comment]
Thanx! In the meantime I tried below approach in helm chart.
It mounts correctly, and the file overrides /usr/local/tomcat/conf/context.xml, but the pod gets crashLoop and dies. And I can't find anything relevant in the logs. Context.xml looks just fine. Any suggestions?

```
extraConfigMap: |-
  context.xml: |
    <Context antiResourceLocking="false" privileged="true" >
      <WatchedResource>WEB-INF/web.xml</WatchedResource>
      <WatchedResource>WEB-INF/tomcat-web.xml</WatchedResource>
      <WatchedResource>${catalina.base}/conf/web.xml</WatchedResource>
      <Resource name="jdbc/postgres_geoserver_read"
          auth="Container"
          type="javax.sql.DataSource"
          driverClassName="org.postgresql.Driver"
          url="jdbc:postgresql://postgis:5432/karta?sslmode=disable"
          username="xxxxxxxxx"
          password="xxxxxxxxx"
          maxTotal="8"
          initialSize="0"
          minIdle="0"
          maxIdle="8"
          maxWaitMillis="-1"
          timeBetweenEvictionRunsMillis="30000"
          minEvictableIdleTimeMillis="60000"
          maxWaitMillis="-1"
          timeBetweenEvictionRunsMillis="30000"
          minEvictableIdleTimeMillis="60000"
          testWhileIdle="true"
          validationQuery="SELECT 1"
          rollbackOnReturn="true"
        />
    </Context>

extraVolume: |-
  - name: geoserver-config-volume
    configMap:
      name: {{ template "geoserver.fullname" . }}

extraVolumeMounts: |-
  - name: geoserver-config-volume
    mountPath: /settings/context.xml
    subPath: context.xml
```

**Comment by NyakudyaA** [comment]
I will test it out as well

**Reply by jesade-vbg** [reply]
Thanx @NyakudyaA !

**Reply by jesade-vbg** [reply]
Hi again. How you hade any time testing to add extraConfigMap the way I did?
